.ci/test.sh

```diff
@@ -48,10 +48,11 @@ if [[ $TASK == "lint" ]]; then
         -c conda-forge \
             libxml2 \
             "r-lintr>=2.0"
-    pip install --user cpplint
+    pip install --user cpplint mypy
     echo "Linting Python code"
     pycodestyle --ignore=E501,W503 --exclude=./.nuget,./external_libs . || exit -1
     pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^external_libs|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
+    mypy --ignore-missing-imports python-package/ || true
```

Comment:
I chose to do `|| true` here instead of `|| exit 0` so it can stay here with other Python package logs